tests/selenium/conftest.py

```diff
@@ -313,6 +313,9 @@ def login(browser: Chrome) -> Chrome:
     browser.find_element(By.ID, login).send_keys("superuser")
     browser.find_element(By.ID, password).send_keys("testtest2")
     browser.find_element(By.XPATH, loginButton).click()
+    from time import sleep
```

Comment:
import should be on top

Reply:
https://github.com/unicef/hope/pull/4679/commits/56b25667077dc8265c69ef2c1182e56ad1ed1cf4